appveyor.yml

```diff
@@ -3,7 +3,7 @@ os:
 
 branches:
   only:
-    - master
```

Comment:
Ideally there would be a way in appveyor.yml to specify the "current target branch".  So if we put in both `master ` and `develop` will that attempt to build a PR meant for develop on master also?  If so, that would likely fail.  Then again, I'm a total AppVeyor n00b.